ai-chat-assistant/README.md

```diff
@@ -0,0 +1,50 @@
+---
+title: AI Chat Assistant
+description: A Slack-based automation assistant that leverages ChatGPT to manage and respond to messages by integrating with GitHub and Google Sheets.
+integrations: ["chatgpt", "github", "googlesheets", "slack"]
```

Comment:
I don't think we support more than 3 - please check with @RonenMars

Reply:
I think we can, there is no limit, but too many might brake the UI